port/linux/ipadapter.c

```diff
@@ -1516,6 +1608,24 @@ oc_connectivity_init(size_t device, oc_connectivity_ports_t ports)
   return 0;
 }
 
+#ifdef OC_DYNAMIC_ALLOCATION
+void
+oc_connectivity_wakeup(void)
```

Comment:
add param `size_t device` and wake only certain device.

Reply:
The method changed to send a wake-up signal to the specific device 
`void oc_connectivity_wakeup(size_t device)`